<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>428</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>428</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>USB-I2C MPU6050</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>角速度</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QwtPlot" name="GyroQwtPlot">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>298</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>角速度</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>X轴：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="angularXLabel">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Sunken</enum>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Y轴：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="angularYLabel">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Sunken</enum>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Z轴：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="angularZLabel">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Sunken</enum>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>加速度</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QwtPlot" name="AccelQwtPlot">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>298</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>加速度</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>X轴：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="acceleratXLabel">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Sunken</enum>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Y轴：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="acceleratYLabel">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Sunken</enum>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>Z轴：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="acceleratZLabel">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::WinPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Sunken</enum>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>温度</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QwtPlot" name="TempQwtPlot">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>298</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>温度</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>温度：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="tempLabel">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::WinPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Sunken</enum>
                   </property>
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>选择设备：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="deviceComboBox">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>NO DEVICE</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>选择通道：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="channelComboBox">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>CH0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CH1</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="initDevicePushButton">
          <property name="text">
           <string>初始化设备</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startPushButton">
          <property name="text">
           <string>开始测量</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="contactUsAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="quitAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="contactUsAction">
   <property name="text">
    <string>联系我们</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
